src/utils/index.js

```diff
@@ -91,22 +91,20 @@ export const blockToDate = (goalBlock, height) => {
   return new Date(diff * 180000 + Date.now());
 };
 
-export const wrapTry = async (promise, store) => {
+// eslint-disable-next-line consistent-return
+export const wrapTry = async (promise) => {
   try {
     return promise.then((res) => {
       if (!res) {
-        if (store) { store.commit('setBackendStatus', false); }
-        return null;
+        throw new Error('No response object');
```

Comment:
In which case there can be no response object? I would remove this check as unnecessary